### ISSUE TITLE
Would be nice if optimize reported child calcjob PK

### ISSUE THREAD

**Issue by DanielMarchand** [greschd/aiida-optimize#18]
If a child calcjob crashes, it would be nice to have its pk reported in the log. Right now it looks like only the CreateEvaluateWorkChain is reported.

**Comment by greschd** [comment]
Agree, I think there are several places where it could be reported:

In `OptimizationWorkChain`: https://github.com/greschd/aiida-optimize/blob/c783fda2992e2e120e3bb5d54733e0a37ee042ef/aiida_optimize/_optimization_workchain.py#L133-L134


In `AddInputsWorkChain`: https://github.com/greschd/aiida-optimize/blob/c783fda2992e2e120e3bb5d54733e0a37ee042ef/aiida_optimize/wrappers/_add_inputs.py#L161-L162

In `CreateEvaluateWorkChain`: https://github.com/greschd/aiida-optimize/blob/c783fda2992e2e120e3bb5d54733e0a37ee042ef/aiida_optimize/wrappers/_create_evaluate.py#L118-L119
